core/cmd/remote_client.go

```diff
@@ -548,3 +555,39 @@ func parseResponse(resp *http.Response) ([]byte, error) {
 	}
 	return b, err
 }
+
+func (cli *Client) checkRemoteBuildCompatibility(lggr logger.Logger, c *clipkg.Context) error {
+	resp, err := cli.HTTP.Get("/v2/build_info")
+	if err != nil {
+		lggr.Warnw("Got error querying for version, remote node version is unknown. CLI may behave in unexpected ways.", "err", err)
+		return nil
+	}
+	b, err := parseResponse(resp)
+	if err != nil {
+		lggr.Warnw("Got error querying for version, remote node version is unknown. CLI may behave in unexpected ways.", "err", err)
```

Comment:
This log message is identical to the one above, but it's actually a different problem. This was an error parsing response, not querying. Can we make the log messages as accurate as possible?